### PR TITLE
Write wikimedia sample test in playwright

### LINKED ISSUE #35: Create a simple test to check for main page heading
1. Go to http://localhost:8080/w
2. Check for main page heading
3. Check if the content for the main page heading loads correctly

### PR DESCRIPTION
**Pull request by hariclerry**
### Tickets
Closes #35 

### What does this PR do?
This PR implements Wikimedia sample test using playwright.

### What are the changes made?
- Write a sample test in playwright for wikimedia.

### Screen shots
None

### CODE REVIEW

**Comment by hariclerry** [comment]
@und3fined-v01d, please check out this PR, it's still a work in progress though.

File: tests/playwright/pageobjects/mainPage.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class MainPage {
```
Comment (sovoid) [inline tests/playwright/pageobjects/mainPage.js]: this should extend a base Page class

**Comment by hariclerry** [comment]
Hey @und3fined-v01d, the test seems to be failing in CI yet it passes locally. Can you please have it at the PR? Thanks

**Comment by hariclerry** [comment]
Hey @und3fined-v01d, the changes have been made. Thanks

**Review by sovoid — APPROVED** [review]

**Pull request merged**